Description: Disable dependency on ansi-to-html
 used in a test to assert ANSI codes in output to stdout, implemented in a
 fragile way.
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -71,4 +71,4 @@
 
-[dev-dependencies.ansi-to-html]
-version = "0.1.3"
+#[dev-dependencies.ansi-to-html]
+#version = "0.1.3"
 
--- a/tests/cli.rs
+++ b/tests/cli.rs
@@ -184,3 +184,3 @@
     // common format, in this case HTML, to check
-    #[test]
+    //#[test]
     fn ambiguous_replace_ensure_styling() {
